examples/OrderEdit/OrderEditAddStagedActionAction.json

```diff
@@ -0,0 +1,9 @@
+{
+  "action": "addStagedAction",
+  "stagedAction": [
+    {
+      "action": "setComment",
```

Comment:
I'm so sorry. setComment is no stageable action. Could you please change it so some action with can be staged? Also I see that the structure only allows one action. So it must be:

```
"stagedAction": {
   "action": "..."
}
```

Reply:
switched to setCountry